tests/h/search/client_test.py

```diff
@@ -37,6 +37,7 @@ def test_conn_is_read_only(self):
     def test_defaults_to_es1(self):
         client = Client(host="http://localhost:9200", index="hypothesis")
 
+        assert client.version == "1.10.0"
```

Comment:
I thought it would be kinda unnecessary to split the verification of the version property into a separate test-we can but I feel like doing it here is fine.